operator/src/main/java/com/datastax/oss/kaap/controllers/bookkeeper/BookKeeperResourcesFactory.java

```diff
@@ -295,9 +295,9 @@ public StatefulSet generateStatefulSet() {
             );
         } else {
             persistentVolumeClaims.add(createPersistentVolumeClaim(journalVolumeName, spec.getVolumes().getJournal(),
-                    labels));
+                    Map.of()));
             persistentVolumeClaims.add(createPersistentVolumeClaim(ledgersVolumeName, spec.getVolumes().getLedgers(),
-                    labels));
+                    Map.of()));
```

Comment:
why the labels replaced with empty map?
If getLabels() returns null in some case let's just fix it and make it return empty map in that case.
In other cases when labels are present, what's the side-effect of not assigning labels? why are we even getting the labels at the beginning of the method?